### ISSUE TITLE
Two PRs created for package in group and separate

### ISSUE THREAD

**Issue by coreyworrell** [dependabot/dependabot-core#8733]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package ecosystem

npm

### Manifest location and content before the Dependabot update

/package.json

### dependabot.yml content

```yml
# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
version: 2
updates:
  - package-ecosystem: npm
    directory: /
    schedule:
      interval: weekly
    ignore:
      - dependency-name: flickity
        versions: ["^3"]
    groups:
      minor:
        update-types:
          - patch
          - minor
```

### Updated dependency

[postcss](https://github.com/postcss/postcss) from 8.4.32 to 8.4.33.

### What you expected to see, versus what you actually saw

Expected to see one PR created as part of the "minor" group. This does happen, but then a separate PR is created for just that single package as well.

### Images of the diff or a link to the PR, issue, or logs

Separate: https://github.com/intervaldev/website/pull/33
Group: https://github.com/intervaldev/website/pull/32

**Comment by jakecoffman** [comment]
It's possible this was fixed by https://github.com/dependabot/dependabot-core/pull/8803.

Have you seen this happen again recently?

**Comment by coreyworrell** [comment]
@jakecoffman I don't think I have seen it happen again.. But I have seen another issue where a PR is created for a non-existent group. In my case it is a group I tested with, then changed my `dependabot.yml` file so it's no longer there. I already did the dependency reevaluation through support too. https://github.com/intervaldev/website/pull/70. I can open new issue for that though if there isn't an existing one for it.
